models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -457,15 +439,9 @@ aeif_cond_beta_multisynapse::calibrate()
   // ensures initialization in case mm connected after Simulate
   B_.logger_.init();
 
-  P_.receptor_types.resize( P_.num_of_receptors );
-  for ( size_t i = 0; i < P_.num_of_receptors; i++ )
-  {
-    P_.receptor_types[ i ] = i + 1;
-  }
-
-  V_.g0_.resize( P_.num_of_receptors );
+  V_.g0_.resize( P_.n_receptors() );
```

Comment:
I think it would be good for readability to point out that `g0_` will be initialized in the loop below.